dev/dll/SharedHelpers.cpp

```diff
@@ -511,6 +511,10 @@ winrt::IconElement SharedHelpers::MakeIconElementFrom(winrt::IconSource const& i
 
         fontIcon.Glyph(fontIconSource.Glyph());
         fontIcon.FontSize(fontIconSource.FontSize());
+        if (const auto newForeground = fontIconSource.Foreground())
```

Comment:
>newForeground [](start = 23, length = 13)

Is this null check needed? setting the foreground to null should be fine right?

Reply:
Unfortunately, without the null check, if the foreground is not specified, you end up with transparent icons as we are overriding the default brush with null.